packages/ui/src/molecules/QuantitySelector/stories/QuantitySelector.stories.tsx

```diff
@@ -0,0 +1,144 @@
+import type { Meta, Story } from '@storybook/react'
+import React, { useState, useMemo } from 'react'
+
+import type {
+  QuantitySelectorProps,
+  QuantitySelectorButtonProps,
+  QuantitySelectorInputProps,
+} from '..'
+import {
+  QuantitySelector,
+  QuantitySelectorButton,
+  QuantitySelectorInput,
+} from '..'
+import { PlusIcon, MinusIcon } from './assets/Icons'
+import mdx from './QuantitySelector.mdx'
+
+const MAX_QUANTITY = 10
+const MIN_QUANTITY = 1
+
+type QuantitySelectorCompoundProps = QuantitySelectorProps &
+  QuantitySelectorButtonProps &
+  QuantitySelectorInputProps
+const QuantitySelectorStylelessTemplate: Story<QuantitySelectorCompoundProps> = (
+  args
+) => {
+  const [quantity, setQuantity] = useState(MIN_QUANTITY)
+
+  function isLeftDisabled() {
+    return quantity === MIN_QUANTITY
+  }
+
+  function isRightDisabled() {
+    return quantity === MAX_QUANTITY
+  }
+
+  return (
+    <QuantitySelector
+      {...args}
+      name="quantity-selector"
+      currentValue={quantity}
+      onClick={(v) => {
+        if (v.currentTarget.name === 'quantity-selector-increase-button') {
+          setQuantity((currentQuantity) =>
+            Math.min(currentQuantity + 1, MAX_QUANTITY)
+          )
+        } else if (
+          v.currentTarget.name === 'quantity-selector-decrease-button'
+        ) {
+          setQuantity((currentQuantity) =>
+            Math.max(currentQuantity - 1, MIN_QUANTITY)
+          )
+        }
+      }}
+    >
+      <QuantitySelectorButton
+        name="quantity-selector-decrease-button"
+        icon={<MinusIcon color="#fff" />}
```

Comment:
```suggestion
        icon={<MinusIcon color="#fff" />}
        aria-label="Decrement"
```

Reply:
btw, we can use `aria-controls` here. We can associate both `QuantitySelectorButton` with `QuantitySelectorInput `. Can you try that, pls? Lemme know if you need help with that :) 

> aria-controls property identifies the element (or elements) whose contents or presence are controlled by the element on which this attribute is set.
Find out more about it: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-controls

Reply:
That's great. Addressed that in https://github.com/vtex/faststore/pull/1035/commits/dfaaee45a47e61f6968277bc4f215e5c6a78bd9c. Lemme know what you think.